tests/infrared/17.0/enable-stf.yaml.template

```diff
@@ -0,0 +1,85 @@
+# enable data collection that is compatible with the STF data model
+# enablement of the collectors and transport are done with separate enviroment files
+# recommended:
+# - environments/metrics/ceilometer-qdr-write.yaml
+# - environments/metrics/collectd-qdr-write.yaml
+# - environments/metrics/qdr-edge-only.yaml
+
+---
+tripleo_heat_templates:
+    []
```

Comment:
I can explain [3]: reason is that the default template strategy is "merge" and so if you include the collectd environment file and then deploy for multi-cloud (or a differently named cloud configuration) you get problems because both the default connector and your newly added connector are configured, so collectd will try to send to an address without any QDR listeners.

Reply:
Thanks guys. Yes Chris, it answers my question.